### ISSUE TITLE
train on IN-1K from scratch with smt_t; 65% top1 of 40 epoches.

### ISSUE THREAD

**Issue by DavideHe** [AFeng-x/SMT#9]
I am not sure I train rightly. 
some setting: 
* opt: adamw
* baselr: 1e-3  warmup first 5 epoch 
* batch_size : 128 
* total epoch: 150 (not 300 because of too long gpu time with 10 days, 150 epoches ,4 T4 gpus)
* lr_scheduler :  cos from 5th epoch
now trained 40 epoches ,65% top1 ,and increase rate slowly  per epoch.
and except total epoch(150) less than default 300;others are almost same as git .
I want to know 2 information:
1、Will  total epoch that set 150 be big influence on final result ?
2、Is it normal that 40 epoches ,65% top1 ? bacause training need long time ,I want to get some information before. If it is wrong,I will stop it early.

**Comment by AFeng-x** [comment]
Hi, we conducted experiments using 8 V100 GPUs. The total batch size was set to 1024, and the training took approximately 2 and a half days. When we trained for 300 epochs, the accuracy of SMT-T at 40 epochs was 71%. If you use 150 epochs and a smaller batch size for training, it may have a significant impact on the final results.

**Comment by DavideHe** [comment]
> Hi, we conducted experiments using 8 V100 GPUs. The total batch size was set to 1024, and the training took approximately 2 and a half days. When we trained for 300 epochs, the accuracy of SMT-T at 40 epochs was 71%. If you use 150 epochs and a smaller batch size for training, it may have a significant impact on the final results.

thanks a lot, I will try more batch_size and trainning epoches with patience.
